execution.bs

```diff
@@ -4108,28 +4111,219 @@ namespace std::execution {
+    that the results of its queries do not depend on the *cv* qualification
+    of the object's type, nor on its value category.
+
+: environment
+:: A queryable object obtainable from a receiver that contains runtime
+    configuration information local to the current operation.
+
+: execution resource
+:: An object, like a thread or a thread pool, that is capable of executing
+    code, and onto which asynchronous operations can be scheduled for
+    execution.
+
+: receiver
+:: An object that receives the results of an asynchronous operation when it
+    completes. It can receive the values on success, an error on failure, or
+    a stopped notification when the operation is cancelled. Additionally,
+    the receiver provides an environment that describes the properties of
+    the 
+
+: operation state
+:: An object responsible for holding the state associated with an asynchronous
+    operation while it is running. That state includes the receiver and the
+    operation states of any nested operations. Operation states are the
+    result of connecting senders and receivers, but they are properly thought
+    of as part of the sender's implementation. Operation states need to be
+    explicitly started.
+
+: sender
+:: An object that encapsulates the description of asynchronous operation,
+    independent of where it will begin execution and what will happen on
+    completion.
+
+: scheduler
+:: A handle to an execution resource that presents a uniform, generic
+    interface for scheduling work onto that resource.
+
+: connect
+:: The act of joining a description of an asynchronous operation -- a
+    sender -- with a consumer of that operation's results -- a receiver.
+    The result of connecting a sender to a receiver is an operation
+    state.
+
+: complete
+:: An asynchronous operation <dfn export=true>completes</dfn> when one of
+    the receiver's completion functions has finished executing. Such a receiver
+    has been completed. A sender completes when the operation state returned
+    from `connect` completes.
+
+: send
+:: When it completes, a sender *sends* its results to a receiver by
+    calling one of the receiver's completion-signal operations,
+    passing the sender's results as arguments.
+
+: completion function
+:: One of `set_value`, `set_error`, or `set_stopped`.
+
+: completion tag
+:: One of `set_value_t`, `set_error_t`, or `set_stopped_t`.
+
+: completion signature
+:: A function type that has one of the following forms:
+    <pre>
+    set_value_t(ValueTypes...)
+    set_error_t(ErrorType)
+    set_stopped_t()</pre>
+
+: completion-signal operation
+:: A valid invocation of a completion function.
+
+: completion scheduler
+:: A scheduler obtainable from a sender's environment that, when present,
+    denotes the scheduler upon which the sender will invoke a completion
+    signal operation. Each completion function may have a different completion
+    scheduler for a particular sender.
```

Comment:
A sender may have a completion scheduler. If present it denotes...

Reply:
A sender may have a different completion scheduler for each completion function.